activerecord/lib/active_record/testing/query_assertions.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  module Testing
+    module QueryAssertions # :nodoc:
```

Comment:
I don't think we should expose this in the `lib` folder. I prefer to duplicate the code than exposing this to the apps. It used to be in the lib folder and we removed years ago.

https://github.com/rails/rails/commit/eb5b7a8ced04383bed0b81ba2ce2194dc3e65629#diff-f44e5c73c8640105ff6fd71298a18302L23.

If all we want is to assert one single query should can't we do in a simpler way?

Reply:
Thanks for the comment, will update 😄